### DISCUSSION TITLE
VIBRATO AUTOMATION

### DISCUSSION THREAD

**Discussion by worker-gray** [SynthstromAudible/DelugeFirmware#2497] [Ideas]
Just like he title describes- I'd like to be able to control the vibrato depth via automation lane as well as the speed (via LFO automation).

**Comment by seangoodvibes** [comment]
You already can

Enter the vibrato menu and then press clip to open automation view for that parameter and then edit the automation as you normally would using the grid

See video:

https://github.com/user-attachments/assets/1b65bc1f-4902-4ae1-a725-fd1b9ed968b4

**Reply by worker-gray** [reply]
OH-- I thought that controlled the LFO-- not the vibe depth. Thanks for clearing that up!

**Comment by seangoodvibes** [comment]
It’s definitely there in Beethoven

**Reply by worker-gray** [reply]
OK-- i had to dig for it but I found it. Do you know if there are plans to make it accessible from the main automation page?

**Comment by seangoodvibes** [comment]
Yea since there’s a grid shortcut for it it should be possible for me to do
that

**Reply by worker-gray** [reply]
You don't have to do it for lil ole ME! I thank you for all the work you've done already! Your contributions have made the best machine even better- and I appreciate you taking the time to direct me to the answer!

**Reply by seangoodvibes** [reply]
> You don't have to do it for lil ole ME! I thank you for all the work you've done already! Your contributions have made the best machine even better- and I appreciate you taking the time to direct me to the answer!

hahah it's really no problem. Just did it. It was on my to do list.
